cms/static/js/views/pages/course_outline.js

```diff
@@ -24,6 +24,9 @@ define(["jquery", "underscore", "gettext", "js/views/pages/base_page", "js/views
                 this.$('.button-new').click(function(event) {
                     self.outlineView.handleAddEvent(event);
                 });
+                this.$('.button.re-index').click(function(event) {
+                    self.outlineView.handleReIndexEvent(event);
```

Comment:
Why is the logic inside the child view? That view represents the rendering of the tree itself and really has nothing to do with indexing. I would prefer to see the method in this page view.